bundles/org.openhab.ui.basic/.gitignore

```diff
@@ -1,6 +1,8 @@
 node_modules
 node
 npm_cache
+src/main/resources/snippets/*
```

Comment:
Maybe *.html ?

Reply:
Not sure if that's a good idea. That means in the future people can start adding non .html files directly there, and we'll back to the situation where there are mixed files (generated + original) in there.

Reply:
If you have a line with *, why do you add a specific line for readme.txt ?

Reply:
It means ignore everything in that directory (because they're generated), but do not ignore (i.e. check in) readme.txt

Reply:
Ok. I did not take attention to the "!" at the beginning of the next line !